packages/legend-application-query/src/stores/query-builder/QueryBuilderConfig.ts

```diff
@@ -25,6 +25,8 @@ export const DEFAULT_POST_FILTER_LAMBDA_VARIABLE_NAME = 'row';
 
 export const QUERY_BUILDER_PROPERTY_SEARCH_MAX_DEPTH = 5;
 export const QUERY_BUILDER_PROPERTY_SEARCH_RESULTS_LIMIT = 100;
+export const QUERY_BUILDER_SEARCH_TEXT_MIN_LENGTH = 3;
```

Comment:
this is not the right place to put it. this file is meant for `QueryBuilder` core, the stuff you're adding is meant for `QueryLoader` which is part of the `Legend Query` app. Please follow my comment in the previous review, these should be put in the same file as `QueryEditorStore` I believe.